Essentials/src/com/earth2me/essentials/commands/Commandtpahere.java

```diff
@@ -35,7 +38,12 @@ public void run(final Server server, final User user, final String commandLabel,
             && player.isTpRequestHere() == true) { // Make sure the last teleport request was actually tpahere and not tpa
             throw new Exception(tl("requestSentAlready", player.getDisplayName()));
         }
+        TPARequestEvent tpaEvent = new TPARequestEvent(user.getSource(), player, true);
         if (!player.isIgnoredPlayer(user)) {
+        	ess.getServer().getPluginManager().callEvent(tpaEvent);
```

Comment:
format again

Reply:
I hope this isn't a line spacing problem - I'll fix it in a future commit if it is, just review this next commit and let me know if it's still there.